src/tasks/runstart.ts

```diff
@@ -479,7 +493,8 @@ export const RunStartQuest: Quest = {
     },
     {
       name: "Use Mind Control Device",
-      completed: () => currentMcd() >= 10 || !canadiaAvailable(),
+      completed: () =>
+        currentMcd() >= 10 || !["platypus", "opossum", "marmot"].includes(mySign().toLowerCase()),
```

Comment:
What's the purpose of this refactoring? It doesn't appear any clearer than before

Reply:
Good call. Reverted.